.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=crlf
```

Comment:
I don't think that it's correct: it's specified in Git documentation: 
> When more than one pattern matches the path, a later line overrides an earlier line. This overriding is done per attribute. 

Source: https://git-scm.com/docs/gitattributes

Nevertheless I'll accept the PR and fix that after ;)